### ISSUE TITLE
Security question: spam through drupalize?

### ISSUE THREAD

**Issue by jangermuller** [lblissett/dn-discuss#24]
Eine komische Sache: auf contact@discourseanalysis.net bekommen wir 2000 error messages (so wie die unten). Haben wir ein security issue mit drupalize-da.de oder discourseanalysis.net? 

The original message was received at Thu, 20 Sep 2018 00:50:35 +0200
from localhost [127.0.0.1]

   ----- The following addresses had permanent fatal errors -----
<souha.hermi@yahoo.fr>
    (reason: 554 delivery error: dd Requested mail action aborted - mta1115.mail.ir2.yahoo.com)

   ----- Transcript of session follows -----
... while talking to mx-eu.mail.am0.yahoodns.net.:
DATA
<<< 554 delivery error: dd Requested mail action aborted - mta1115.mail.ir2.yahoo.com
554 5.0.0 Service unavailable
Reporting-MTA: dns; www.drupalize-da.de
Arrival-Date: Thu, 20 Sep 2018 00:50:35 +0200

Final-Recipient: RFC822; souha.hermi@yahoo.fr
Action: failed
Status: 5.0.0
Remote-MTA: DNS; mx-eu.mail.am0.yahoodns.net
Diagnostic-Code: SMTP; 554 delivery error: dd Requested mail action aborted - mta1115.mail.ir2.yahoo.com
Last-Attempt-Date: Thu, 20 Sep 2018 11:30:09 +0200
Return-Path: <contact@discourseanalysis.net>
Received: from www.drupalize-da.de (localhost [127.0.0.1])
	by www.drupalize-da.de (8.15.2/8.15.2/Debian-3) with ESMTP id w8JMoZ8u031951
	for <souha.hermi@yahoo.fr>; Thu, 20 Sep 2018 00:50:35 +0200
Received: from [IPv6:::1] (www-data@localhost)
	by www.drupalize-da.de (8.15.2/8.15.2/Submit) with SMTP id w8JMoZVJ031950
	for <souha.hermi@yahoo.fr>; Thu, 20 Sep 2018 00:50:35 +0200
X-Authentication-Warning: www.drupalize-da.de: www-data owned process doing -bs
Sender: contact@discourseanalysis.net
Message-ID: <8e12cd2d1a5cb2c84315e40e1c4d9371@localhost>
Date: Thu, 20 Sep 2018 00:50:35 +0200
Subject: DN newsletter: The Discursive Construction of Reality IV -
Interdisciplinary perspectives on a  sociology of knowledge approach to
discourse research
From: mailer@discourseanalysis.net
Reply-To: mailer@discourseanalysis.net
To: souha.hermi@yahoo.fr
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable
X-Mailer: Drupal

**Comment by jangermuller** [comment]
Ich bekomme weiter error message auf contact@d..., die moeglicherweise auf Drupal zurueckegehen, z.b. gerade das hier. Was ist da wohl los? 

The original message was received at Thu, 20 Sep 2018 00:54:01 +0200
from localhost [127.0.0.1]

   ----- The following addresses had permanent fatal errors -----
<sandyhaifeng@yahoo.com.cn>
    (reason: 550 relaying denied for <sandyhaifeng@yahoo.com.cn>)

   ----- Transcript of session follows -----
... while talking to mta6.am0.yahoodns.net.:
DATA
<<< 550 relaying denied for <sandyhaifeng@yahoo.com.cn>
550 5.1.1 <sandyhaifeng@yahoo.com.cn>... User unknown
<<< 504 At least one RCPT command is required
Reporting-MTA: dns; www.drupalize-da.de
Arrival-Date: Thu, 20 Sep 2018 00:54:01 +0200

Final-Recipient: RFC822; sandyhaifeng@yahoo.com.cn
Action: failed
Status: 5.1.1
Remote-MTA: DNS; mta6.am0.yahoodns.net
Diagnostic-Code: SMTP; 550 relaying denied for <sandyhaifeng@yahoo.com.cn>
Last-Attempt-Date: Thu, 20 Sep 2018 19:30:13 +0200
Return-Path: <contact@discourseanalysis.net>
Received: from www.drupalize-da.de (localhost [127.0.0.1])
	by www.drupalize-da.de (8.15.2/8.15.2/Debian-3) with ESMTP id w8JMs1g6003473
	for <sandyhaifeng@yahoo.com.cn>; Thu, 20 Sep 2018 00:54:01 +0200
Received: from [IPv6:::1] (www-data@localhost)
	by www.drupalize-da.de (8.15.2/8.15.2/Submit) with SMTP id w8JMs1DE003472
	for <sandyhaifeng@yahoo.com.cn>; Thu, 20 Sep 2018 00:54:01 +0200
X-Authentication-Warning: www.drupalize-da.de: www-data owned process doing -bs
Sender: contact@discourseanalysis.net
Message-ID: <a36e7d806564ea0d8cada82e5a475d0f@localhost>
Date: Thu, 20 Sep 2018 00:54:00 +0200
Subject: DN newsletter: The Discursive Construction of Reality IV -
Interdisciplinary perspectives on a  sociology of knowledge approach to
discourse research
From: mailer@discourseanalysis.net
Reply-To: mailer@discourseanalysis.net
To: sandyhaifeng@yahoo.com.cn
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable
X-Mailer: Drupal

**Comment by lblissett** [comment]
Ja, danke für die Rückmeldung. Das müssen wir mal nachprüfen, womit das zusammenhängt. 

Allgemein wäre es gut den Email-Versand an einen anderen Service auszulagern. Wenn man Mails von seinem eigenen Server verschickt und etwas nicht korrekt läuft, dann wird man von vielen E-Mail Provider schnell als Spammer deklariert. Diesem Punkt würde ich eine hohe Priorität geben um solche Probleme in Zukunft zu vermeiden. 

Wie schon hier diskutiert: #17

**Comment by jangermuller** [comment]
Ok, danke! Wir muessten am besten morgen eine grosse Rundmail an alle machen, dass dnc3aled.discourseanalysis.net wieder funktioniert!

**Comment by jangermuller** [comment]
Wir das gehen?

**Comment by bits-of-static** [comment]
Wir können es machen, aber ich wurde davon abraten.

Die Umstellung auf Amazon SES hat Priorität, da wir genau solche Probleme in eine kontrollierte Umgebung beheben können. Wir können dieses eine Problem suchen und beheben, aber wir können nicht versichern dass nichts neues aufkommt. Das wes zur zeit läuft ist eine ersatz Lösung.

**Comment by bits-of-static** [comment]
Außerdem, mit jedem Fehlversuch, rutscht man näher zu den spam ordern.

**Comment by jangermuller** [comment]
Wenn wir dann umgehend auf amazon umsteigen?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by bits-of-static** [comment]
Ich wurde es gern Anfang nächste Woche so weit haben dass alle den emailing ohne sorgen nutzen können.

**Comment by jangermuller** [comment]
Kannst du einschaetzen, ob dieser Spam an viele oder gar alle geht? Wenn es nur contact@d... ist, dann waere es Ok. Ansonsten waere jede Stunde gefaehrlich. Wir bekommen jeden Tag 100 von diesen  messages.

**Comment by bits-of-static** [comment]
Das ist kein spam, das ist nur info nachricht an uns. Die sehen nur wir.

**Comment by jangermuller** [comment]
Uff, gut!
